core/swift3.1Action/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Dockerfile for swift actions, overrides and extends ActionRunner from actionProxy
+# This Dockerfile is partially based on: https://github.com/IBM-Swift/swift-ubuntu-docker/blob/master/swift-development/Dockerfile
+FROM ibmcom/swift-ubuntu:3.1
+
+ENV SWIFT_LOCATION /root/swift-3.1-RELEASE-ubuntu14.04
```

Comment:
Can we please use `swift-3.1.1-RELEASE-ubuntu14.04` instead, since it's out now? It fixes some bugs that were in 3.1 that could cause problems for people.